ecosystem/sf-indexer/aptos-sf-indexer/src/database.rs

```diff
@@ -51,11 +51,11 @@ where
     <T as diesel::QuerySource>::FromClause: diesel::query_builder::QueryFragment<diesel::pg::Pg>,
 {
     let debug = diesel::debug_query::<diesel::pg::Pg, _>(&query).to_string();
-    aptos_logger::debug!("Executing query: {:?}", debug);
+    aptos_logger::debug!(debug = debug, "Executing query");
```

Comment:
this should be named `query = debug` instead of just debug

Reply:
lemme try.